NETWORK/N_0xFEA7A352DDB34D52.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: NETWORK 
+---
+
+## _0xFEA7A352DDB34D52
+
+```c
+// 0xFEA7A352DDB34D52 
+void _0xFEA7A352DDB34D52(Any p0);
```

Comment:
Seems to be named `NETWORK_BLOCK_PROXY_MIGRATION_BETWEEN_TUTORIAL_SESSIONS` with param BOOL if block or not. 

This native seems to prevents the transfer of network objects between players situated in different tutorial instances. I guess the flag needs to be deactivated when its purpose is fulfilled or no longer required.